cmd/fossa/cmd/test/test.go

```diff
@@ -28,6 +29,7 @@ var Cmd = cli.Command{
 	Action: Run,
 	Flags: flags.WithGlobalFlags(flags.WithAPIFlags(([]cli.Flag{
 		cli.IntFlag{Name: Timeout, Value: 10 * 60, Usage: "duration to wait for build completion (in seconds)"},
+		cli.BoolFlag{Name: NoFail, Usage: "don't exit with error if issues are found"},
 	}))),
 }
 
```

Comment:
Github won't let me comment on the lines directly, but we should consider if we want to be suppressing any of the errors from fossa not failing on line 41 and 46. I think I agree with ignoring these `Fatal` exits like you have, which pushes me towards making the flag more specific.